controllers/repo_manager/utils.go

```diff
@@ -144,27 +148,57 @@ func getPulpSetting(pulp *repomanagerpulpprojectorgv1beta2.Pulp, key string) str
 
 }
 
+// sortKeys will return an ordered slice of strings defined with the keys from a.
```

Comment:
from a?

Reply:
"_a_" is the parameter from `sortKeys` :/